### DISCUSSION TITLE
viewswitch hotkeys

### DISCUSSION THREAD

**Discussion by Zoltan45** [mamedev/mame-goodies#3] [Q&A] [answered]
Can the viewswitch plugin be made to use one hotkey to cycle through available views? @cuavas

**Comment by cuavas** [comment, marked as answer]
Check the updated plugin after the changes in 63dd8a6c98fd04d32c949e5cdd64e2afd800c3ca.  That allows you to set hotkeys to cycle through available view.  The hotkeys are saved per-system.

**Reply by Zoltan45** [reply]
works great. thank you very much!
